engine/common/components/src/main/scala/pl/touk/nussknacker/engine/common/components/DecisionTable.scala

```diff
@@ -92,14 +94,14 @@ object DecisionTable extends EagerService with SingleInputDynamicComponent[Servi
         implicit ec: ExecutionContext,
         collector: InvocationCollectors.ServiceInvocationCollector,
         componentUseCase: ComponentUseCase
-    ): Future[Any] = Future {
+    ): Future[DecisionTableResponse] = Future {
```

Comment:
there is no sense in changing type here

Reply:
Hmm, I know that it is not propagated further but it forces checks on this type with whats inside Future, it can show that we've changes `filterRows` method to, let's say, Map[Int, Boolean] - with `Any` it will just quietly pass